app/assets/javascripts/hyrax/app.js

```diff
@@ -134,5 +135,12 @@ Hyrax = {
     statistics: {},
 
     // initialized in hyrax/config.js
-    config: {}
+    config: {},
+
+    // Per Page select will submit its form to change records shown
+    batchSubmit: function () {
+        var BatchSubmit = require('hyrax/batch_select');
```

Comment:
Yeah, we're just ignoring these. I guess eventually we could convert app.js to app.es6 and do away with these altogether.